bin/cron/analyze_hoc_activity

```diff
@@ -76,11 +76,13 @@ def main
     day += 1
   end
 
+  hoc_year = DCDO.get("hoc_year", 2017)
```

Comment:
Thanks.  Also found another similar case.  @tanyaparker and I are scheduled to test all of this early next week to make sure it's working as expected, before merging.